### PR TITLE
update: text edits from Swedish to English

### PR DESCRIPTION
**Pull request by EugeneJoe**

### CODE REVIEW

File: sb.web.app/src/components/CreateArticle/NewArticle.vue
Comment (utunga) [inline sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: This is a misspelling? Huh?

File: sb.web.app/src/components/CreateArticle/StepTwo.vue
Comment (utunga) [inline sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: fwiw the i18n library has special functions for doing this kind of thing... that would let us keep all that logic in the translation side of things..

File: sb.web.app/src/components/CreateArticle/NewArticle.vue
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: I guess just a typo that slipped through.

File: sb.web.app/src/components/CreateArticle/StepTwo.vue
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific 

Also if you want to access the $i18n t method from code (as in not in the vue template) you can do it this way 
`this.$i18n.t('time.now')`

File: sb.web.app/src/components/Navbar/Notifications.vue
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:36]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:54]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:83]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:99]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Profile/MyProfile.vue
Comment (utunga) [inline sb.web.app/src/components/Profile/MyProfile.vue:50]: Sorry but this needs to be done via i18n not hardcoded to English

File: sb.web.app/src/locales/en.json
Comment (utunga) [inline sb.web.app/src/locales/en.json:156]: Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that can avoid these weird 'sentence fragments' lying around

**Review by utunga — CHANGES_REQUESTED** [review]
Marking this as request changes - can't merge as is sorry

File: sb.web.app/src/components/CreateArticle/NewArticle.vue
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: If you can fix it back to how it was then come back

File: sb.web.app/src/locales/en.json
Comment (EugeneJoe) [reply sb.web.app/src/locales/en.json:156]: Okay.
I figured we were doing a full switch to English.
I'll chuck them into the en.josn and use that for all these scenarios

Comment (EugeneJoe) [reply sb.web.app/src/locales/en.json:156]: We'll kind of have to hard-code some of these values in the code if we end up using this named formatting method.

File: sb.web.app/src/components/CreateArticle/StepTwo.vue
Comment (EugeneJoe) [reply sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: Won't we end up chucking translations into the html code if I use named formatting? @utunga

File: sb.web.app/src/locales/se.json
Comment (utunga) [inline sb.web.app/src/locales/se.json:124]: are these keys also in en.json ?

**Review by utunga — CHANGES_REQUESTED** [review]
Still a couple more changes wanted.. i sent as telegram comment

File: sb.web.app/src/locales/se.json
Comment (EugeneJoe) [reply sb.web.app/src/locales/se.json:124]: yes.
Added them in both

**Review by utunga — APPROVED** [review]
Approving so we can get these changes deployed. It's better like this than not. 

But we don't want *any* English *or* Swedish text in the code if we can avoid it.

**Pull request merged**
